.btcdeb_history

```diff
@@ -0,0 +1,4 @@
+setp
```

Comment:
What is this file?  Is it required by Travis?

Reply:
oh no this is from running `btcdeb` in this folder. I should remove it.